src/injected/recorder.ts

```diff
@@ -155,7 +155,7 @@ export class Recorder {
       return;
     if ((event.target as Element).nodeName === 'INPUT') {
       // Check/uncheck are handled in input.
-      if (((event.target as HTMLInputElement).type || '').toLowerCase() === 'checkbox')
+      if (['checkbox', 'date'].includes(((event.target as HTMLInputElement).type || '').toLowerCase()))
```

Comment:
Use `shouldIgnoreMouseEvent` from below, then handle check as it used to be

Reply:
yeah also thought about that. Test broke then didn't do it in the end. Now after adjusting the test to be more realistic, it works.